doc/source/workflows/management.rst

```diff
@@ -82,12 +83,22 @@ Storage Configuration
 Workflows supports two types of storage backends out of the box:
 
 *  Local file system: the data is stored locally. This is only for single node testing. It needs to be a NFS to work with multi-node clusters. To use local storage, specify ``ray.init(storage="/path/to/storage_dir")``.
-*  S3: Production users should use S3 as the storage backend. Enable S3 storage with ``r.init(storage="s3://bucket/path")``.
+*  S3: Production users should use S3 as the storage backend. Enable S3 storage with ``ray.init(storage="s3://bucket/path")``.
 
 Additional storage backends can be written by subclassing the ``Storage`` class and passing a storage instance to ``ray.init()`` [TODO: note that the Storage API is not currently stable].
 
 If left unspecified, ``/tmp/ray/workflow_data`` will be used for temporary storage. This default setting *will only work for single-node Ray clusters*.
 
+Concurrency Control
+-------------------
+Ray workflow supports concurrency control. You can support the maximum running workflows and maximum pending workflows via ``workflow.init()``
+before executing any workflow. ``workflow.init()`` again with a different configuration would raise an error.
+
+For example, ``workflow.init(max_running_workflows=10, max_pending_workflows=50)`` means there will be at most 10 workflows running, 50 workflows pending.
+Submitting workflows when the number of pending workflows are at maximum would raise ``queue.Full("Workflow queue has been full")``. Getting the output of a pending workflow would be blocking until the workflow finishes running later.
+
+Pending workflows have the ``PENDING`` status, they can be resumed same as these running workflows if they failed with a system error.
```

Comment:
What does this mean?

Reply:
I mean if a workflow status is `RUNNING` and the workflow failed, we can resume the workflow. Same as the `PENDING` status.